datahub-web-react/src/graphql/dataFlow.graphql

```diff
@@ -125,6 +125,53 @@ query getDataFlow($urn: String!) {
     }
 }
 
+query getDataFlowChildJobs($urn: String!, $start: Int, $count: Int) {
```

Comment:
nice! as a followup we can actually remove the childJobs that we're currently getting in the main dataflow query since we're getting it now when you open the tab

Reply:
As we're utilizing the 'count' field from the 'childJobs'  in other parts of our main dataflow query, I propose removing fields that aren't utilized elsewhere from the main query.